cmd/mayactl/app/command/stats_helper.go

```diff
@@ -0,0 +1,159 @@
+package command
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+// Volume is a command implementation struct
+type Volume struct {
```

Comment:
@utkarshmani1997  Forget to inform you , please use the `Volume{}` struct from the `types.go` files.

Reply:
@prateekpandey14 signature of both structures are different and there is some cleanup we have to do later regarding this.

Reply:
ok